src/pages/zh-cn/editor-setup.md

```diff
@@ -20,19 +20,62 @@ Astro 可以和任意编辑器一同工作。不过我们推荐在 Astro 项目
 
 开始前需要先安装 [Astro VS Code 扩展](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)。
 
->⚙️ 查看如何在你的 Astro 项目中[设置 TypeScript](/zh-cn/guides/typescript/)。
+📚 查看如何在你的 Astro 项目中[设置 TypeScript](/zh-cn/guides/typescript/)。
 
 ## 其他代码编辑器
 
 我们令人惊喜的社群为其他受欢迎的编辑器维护了几个扩展，它们包括：
 
 - [Open VSX 上的 VS Code 扩展](https://open-vsx.org/extension/astro-build/astro-vscode) <span style="margin: 0.25em;"><Badge variant="accent">官方</Badge></span> - 官方 Astro VS Code 扩展, 同时在 Open VSX registry 上分发，为其他开发平台如 [VSCodium](https://vscodium.com/) 提供支持
-- [Nova 扩展](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/)<span style="margin: 0.25em;"><Badge variant="neutral">社群</Badge></span> - 为 Astro 提供语法高亮,、智能提示、自动补全
+- [Nova 扩展](https://extensions.panic.com/extensions/sciencefidelity/sciencefidelity.astro/)<span style="margin: 0.25em;"><Badge variant="neutral">社区</Badge></span> - 为 Astro 提供语法高亮,、智能提示、自动补全
+- Neovim [LSP](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#astro) 和 [TreeSitter](https://github.com/virchau13/tree-sitter-astro) 插件 <span style="margin: 0.25em;"><Badge variant="neutral">社区</Badge></span> - 为 Neovim 内的 Astro 提供语法高亮、treesitter 解析和代码补全。
+
+### JetBrains IDE
+
+我们很想支持 [Webstorm IDE](https://www.jetbrains.com/webstorm/)。不幸的是，它不支持语言服务器，而且我们没有足够的资源来编写并维护一个与我们现有代码库语言不同的完全独立的扩展。请访问相关的 [JetBrains 支持问题](https://youtrack.jetbrains.com/issue/WEB-52015/Astro-Language-Support)，对工单进行投票，跟踪进展，并找到社区解决方法。
+
+然而，JetBrains 即将推出的 [Fleet IDE](https://www.jetbrains.com/fleet/) 将支持语言服务器，我们目前可用的工具将能够毫无障碍地在那里运行.
 
 ## 云端编辑器
 
 除本地编辑器外，Astro 同样适用于云端托管编辑器，包括：
 
 - [StackBlitz](https://stackblitz.com) 和 [CodeSandbox](https://codesandbox.io) - 运行在你浏览器中的编辑器，并为 `.astro` 文件提供支持并内置语法高亮。无需安装或配置！
 - [GitHub.dev](https://github.dev) - 可以将 Astro VS Code 扩展安装为[网页 extension](https://code.visualstudio.com/api/extension-guides/web-extensions) 只能使用部分功能，目前仅支持语法高亮。
-- [Gitpod](https://gitpod.io) - 云上的完整开发环境，可以从 Open VSX 上安装官方 Astro VS Code 扩展。 
+- [Gitpod](https://gitpod.io) - 云上的完整开发环境，可以从 Open VSX 上安装官方 Astro VS Code 扩展。
+
+## 其他工具
+
+### ESLint
+
+[ESLint](https://eslint.org/) 是流行的 JavaScript 和 JSX 的 linter。为了支持 Astro，需要安装[一个由社区维护的插件](https://github.com/ota-meshi/eslint-plugin-astro)。
+
+关于如何为你的项目安装和设置 ESLint 的更多信息，请参见[the project's User Guide](https://ota-meshi.github.io/eslint-plugin-astro/user-guide/)。
```

Comment:
There's some leftover English text here

Reply:
This is deliberate, as the original title is in English.